setup/prul.ml

```diff
@@ -0,0 +1,115 @@
+let get_some = function
```

Comment:
I figured out to do both: repl fiddling & separate files. Feel free to suggest a better name for 'stuff that we use more than once, and doesn't really fit anywhere else'